packages/react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -230,20 +235,16 @@ export class DetailsHeaderBase
                     hostClassName: classNames.checkTooltip,
                     id: `${this._id}-checkTooltip`,
                     setAriaDescribedBy: false,
-                    content: ariaLabelForSelectAllCheckbox,
+                    content: checkboxLabel,
                     children: (
                       <DetailsRowCheck
                         id={`${this._id}-check`}
-                        aria-label={
-                          selectionMode === SelectionMode.multiple
-                            ? ariaLabelForSelectAllCheckbox
-                            : ariaLabelForSelectionColumn
-                        }
+                        aria-label={checkboxLabel}
                         data-is-focusable={!isCheckboxHidden || undefined}
                         isHeader={true}
                         selected={isAllSelected}
                         anySelected={false}
-                        canSelect={!isCheckboxHidden}
+                        canSelect={selectionMode === SelectionMode.multiple ? !isCheckboxHidden : !!isAllSelected}
```

Comment:
I'm worried a bit about this line, since now it's ignoring `selectAllVisibility` for everything but `SelectionMode.multiple` (which I assume is the reason for the screener difference). Would it make sense as this instead?

`canSelect={selectionMode === SelectionMode.multiple ? !isCheckboxHidden : !isCheckboxHidden && !!isAllSelected}`

Reply:
@ThomasMichon What do you think of Sarah's suggestion?

Reply:
@ThomasMichon Friendly bump 3.  :)